addons/mail/static/src/core/common/message_model.js

```diff
@@ -71,7 +71,7 @@ export class Message extends Record {
                         : record_name || undefined,
             });
         }
-        assignIn(this, data, ["author", "notifications", "reactions", "recipients"]);
+        assignIn(this, data, ["author", "author_id", "notifications", "reactions", "recipients"]);
```

Comment:
Better adapt formatter in python to receive `author: { id, type: "partner" }`. We want JS model data insertion to be trivial with no custom code in this `update` method eventually. To do so, the data should be formatted in a way to match JS model fields